install/rdtclient-install.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2023 tteck
+# Author: tteck (tteckster)
+# License: MIT
+# https://github.com/tteck/Proxmox/raw/main/LICENSE
+
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y curl
+$STD apt-get install -y sudo
+$STD apt-get install -y mc
+$STD apt-get install -y unzip
+msg_ok "Installed Dependencies"
+
+msg_info "Installing rdtclient"
+wget -q https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb
```

Comment:
Why use an Ubuntu package instead of a Debian package?
`wget -q https://packages.microsoft.com/config/debian/12/packages-microsoft-prod.deb`

Reply:
Just followed the instructions on the rdtclient repo, if there’s a Debian
one, it’s easy enough to test